Dockerfile

```
@@ -1,11 +1,37 @@
-FROM python:3-slim
+FROM docker.io/python:3.7-alpine
 LABEL maintainer="Vladislav Yarmak <vladislav-ex-src@vm-0.com>"
 
-COPY . /build
+ARG UID=18721
+ARG USER=mta-sts
+ARG GID=18721
+
+RUN true \
+   && addgroup --gid "$GID" "$USER" \
+   && adduser \
+        --disabled-password \
+        --gecos "" \
+        --home "/build" \
+        --ingroup "$USER" \
+        --no-create-home \
+        --uid "$UID" \
+        "$USER" \
+   && true
+
+COPY --chown=mta-sts:mta-sts . /build
```

```suggestion
COPY . /build
```
Older docker versions do not recognize this flag and fail with error `Unknown flag: chown`. We don't need it anyway since `USER` directive is one of last steps and image builds just fine without that.

---

Agreed, since we do a chown anyway. would just reduce the delta.